gloo/transport/unbound_buffer.h

```diff
@@ -98,7 +98,7 @@ class UnboundBuffer {
   // If the byte count argument is not specified, it will default the
   // number of bytes to be equal to the number of bytes remaining in
   // the buffer w.r.t. the offset.
-  static constexpr auto kUnspecifiedByteCount = std::numeric_limits<size_t>::max();
+  static constexpr auto kUnspecifiedByteCount = (std::numeric_limits<size_t>::max)();
```

Comment:
`#define NOMINMAX 1` wherever `Windows.h` is being included, or preferably at the build system level. No reason to have to fix it at every call site.